src/containers/org/members/OrgMembersContainer.js

```diff
@@ -94,117 +54,20 @@ const OrgMembersContainer = ({ organization, organizationId } :Props) => {
 
   const dispatch = useDispatch();
 
-  const [addOrRemoveMemberRoleAction, setAddOrRemoveMemberRoleAction] = useState();
-  const [addOrRemoveOrgMemberAction, setAddOrRemoveOrgMemberAction] = useState();
-  const [searchQuery, setSearchQuery] = useState();
-  const [targetMember, setTargetMember] = useState();
-  const [targetRole, setTargetRole] = useState<Role | void>();
+  const [selectedRole, setSelectedRole] = useState();
 
   const getOrganizationMembersRS :?RequestState = useRequestState([ORGANIZATIONS, GET_ORGANIZATION_MEMBERS]);
-  const searchAllUsersRS :?RequestState = useRequestState([USERS, SEARCH_ALL_USERS]);
 
   const isOwner :boolean = useSelector((s) => s.getIn([ORGANIZATIONS, IS_OWNER, organizationId]));
```

Comment:
just curious, where does this isOwner get set? i would like to do something similar for CARE

Reply:
https://github.com/openlattice/lattice-orgs/blob/develop/src/containers/orgs/sagas/getOrganizationsAndAuthorizations.js